rest/zone.go

```diff
@@ -55,6 +65,32 @@ func (s *ZonesService) Get(zone string) (*dns.Zone, *http.Response, error) {
 	return &z, resp, nil
 }
 
+// gets and appends another list of zones to the passed list
+func (s *ZonesService) nextZones(v *interface{}, uri string) (*http.Response, error) {
+	tmpZl := []*dns.Zone{}
+	resp, err := s.client.getURI(&tmpZl, uri)
+	if err != nil {
+		return nil, err
+	}
+	zoneList := (*v).(*[]*dns.Zone)
```

Comment:
yeah this is not the cleanest or most maintainable approach, but I think we wanna stick with it for now. Longer term I think we'd like to advocate/discuss some API changes that would allow for finer-grained pagination control for users, without exposing them to confusing implementation details.